js/impress.js

```diff
@@ -724,24 +724,31 @@
         //   Well, the [tab] key by default navigates around focusable elements, so clicking
         //   it very often caused scrolling to focused element and breaking impress.js
         //   positioning. I didn't want to just prevent this default action, so I used [tab]
-        //   as another way to moving to next step... And yes, I know that for the sake of
-        //   consistency I should add [shift+tab] as opposite action...
+        //   as another way to moving to next step...
         document.addEventListener( "keyup", function( event ) {
 
+            // Handles Tab and Shift + Tab
+            if ( event.keyCode === 9 ) {
+              if ( event.shiftKey ) {
+                api.prev();
+              } else {
+                api.next();
+              }
+              return;
+            }
+
             if ( event.shiftKey || event.altKey || event.ctrlKey || event.metaKey ) {
```

Comment:
Those are the ones who seems to be related to this:

* https://github.com/impress/impress.js/issues/127
* https://github.com/impress/impress.js/issues/238
* https://github.com/impress/impress.js/issues/304
* https://github.com/impress/impress.js/issues/384

I will take a look in those and try to reproduce it.

I wonder if it's worth keeping this issue in 0.6.0 milestone though, given we don't have a single test to back this up yet. It seems more complex than I thought and maybe it is not worth the risk of landing without tests for something that is even hard to reproduce.